src/formats/apple-notes.ts

```diff
@@ -0,0 +1,346 @@
+import { Notice, Platform, Setting, TAbstractFile, TFile, TFolder } from 'obsidian';
+import { NoteConverter } from './apple-notes/convert-note';
+import { ANAccount, ANAttachment, ANConverter, ANConverterType, ANFolderType } from './apple-notes/models';
+import { descriptor } from './apple-notes/descriptor';
+import { ImportContext } from '../main';
+import { fs, os, path, splitext } from '../filesystem';
+import { FormatImporter } from '../format-importer';
+import { ungzip } from 'pako';
+import { Root } from 'protobufjs';
+import SQLiteTagSpawned from 'sqlite-tag-spawned';
+
+const NOTE_FOLDER_PATH = 'Library/Group Containers/group.com.apple.notes';
+const NOTE_DB = 'NoteStore.sqlite';
+/** Additional amount of seconds that Apple CoreTime datatypes start at, to convert them into Unix timestamps. */
+const CORETIME_OFFSET = 978307200; 
+
+const ROOT_DOC = Root.fromJSON(descriptor);
```

Comment:
Can we move this to inside the importer, so we aren't initializing it for every other importer type?